### DISCUSSION TITLE
Video Playback Speed Controller

### DISCUSSION THREAD

**Discussion by Erich-Chen** [filebrowser/filebrowser#1018] [Ideas]
May I request the support of adjusting video playback speed, please? 
It is often useful to play a video in 0.75x, 1.25x, 1.5x, 2x, etc.

**Comment by Silver911r** [comment]
This would be really great.  People speak slower than I would like in most tutorials.

I found a work around that seems good enough.  Because filebrowser is using embed video tag, install an extension that allows you to speed up embedded videos.  I'm using "accelerate" for safari and there are other extensions for chrome and firefox.
